singularity-eos/eos/eos_davis.hpp

```diff
@@ -292,6 +292,7 @@ class DavisProducts : public EosBase<DavisProducts> {
   inline void Finalize() {}
   static std::string EosType() { return std::string("DavisProducts"); }
   static std::string EosPyType() { return EosType(); }
+  // TODO (JHP): Create helper function to find the CJ state given the reference state
```

Comment:
:+1: I'm in favor of the helper being `static` too.

Reply:
Given that we now use the shift to set the energy of the CJ state, I am doubly convinced we should be providing a method to compute it, so that the shift can be straightforwardly generated. I suppose it doesn't need to be static necessarily... since you could generate an EOS, compute the energy, then shift it.

Reply:
Yeah... since the modifier takes an initialized object, it seems most prudent to have it _not_ be static so that you don't have to copy/paste parameters and risk an error.

We might want to discuss what this should look like on the Fortran side though. I'll make a comment in the issue.